src/prefect/utilities/jira_notification.py

```diff
@@ -0,0 +1,121 @@
+"""
+A state-handler that will create and assign a Jira ticket. 
+"""
+
+from typing import TYPE_CHECKING, Union, cast
+from datetime import datetime
+from toolz import curry
+
+import prefect
+
+
+try:
+    from jira import JIRA
+except ImportError:
+    raise ImportError(
+        'Using `jira_notifier` requires Prefect to be installed with the "jira" extra.'
+    )
+
+if TYPE_CHECKING:
+    import prefect.engine.state
+    import prefect.client
+    from prefect import Flow, Task
```

Comment:
One of our CI checks is failing because of these unused imports; I recommend removing this line and replacing

```python
TrackedObjectType = Union[prefect.Flow, prefect.Task]
```

below

Reply:
A ha!  Thank you!